os/net/bluetooth/bt_conn.c

```diff
@@ -626,6 +677,30 @@ FAR struct bt_conn_s *bt_conn_lookup_addr_le_internal(FAR const bt_addr_le_t *pe
 	return NULL;
 }
 
+struct bt_conn_s *bt_conn_lookup_addr_le_id(uint8_t id, const bt_addr_le_t *peer)
```

Comment:
Same as above, please move this functionality in 'bt_conn_lookup_addr_le'.

Reply:
We are using bt_conn_lookup_addr_le_id() at some other places also.